MainRepository/Program.cs

```diff
@@ -1,7 +1,27 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find user is supposed to guess a number between 0 and 100 in a maximum of 8 attempts.
+namespace Test2 {
+   class Program {
+      static void Main (string[] args) {
+         int rand = new Random ().Next (1, 101), tryCount = 0;
+         while (tryCount < 8) {
+            Console.WriteLine ("Enter your guess (1-100): ");
+            if (!int.TryParse (Console.ReadLine (), out int number) || number < 0) {
+               Console.WriteLine ("Enter the valid input");
+            } else {
+               tryCount++;
+               var x = rand.CompareTo (number) switch {
+                  1 => "The number is higher, guess again.",
+                  -1 => "The number is lesser, guess again.",
+                  _ => "Correct, you Win!",
+               };

```

Comment:
See https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/patterns#relational-patterns and change the code to not use CompareTo method.